### PR TITLE
get the checklist on a card implemented

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: trello/jsonfiles/checklist_schema_array.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "id",
+        "name",
+        "idBoard",
+        "idCard",
+        "pos",
+        "checkItems"
+      ]
+    }
+  ]
+}
```
Comment (Enrique-C) [inline trello/jsonfiles/checklist_schema_array.json]: At a code line at the end

**Review by Enrique-C — APPROVED** [review]

File: trello/api/features/negative/BoardNegative.feature [hunk 2]
```diff
@@ -8,24 +7,19 @@ Feature: Negative test of Board
...
-      | name |                     |
-      | desc | This is description |
-    And Sends request
-    And Should return status code 400
-    And Validates response message with message "invalid value for name"
-
-
-  Scenario: Creates a new board with a name empty
+  @defect
+  Scenario Outline: Create a new board with spaces as name
```
Comment (Amet015) [inline trello/api/features/negative/BoardNegative.feature]: createS

Comment (Amet015) [inline trello/api/features/negative/BoardNegative.feature]: Please put all the the scenario in negative

File: trello/api/features/functional/CardFuntional.feature [hunk 3]
```diff
@@ -287,3 +287,54 @@ Feature: Card
...
+    And Validates schema with "checklist_schema_array.json"
+    And Sets a "GET" request to "/checklists/ChecklistObject.id"
+      | key    | value |
+      | fields | all   |
+      | cards  | all   |
+    And Sends request
+    And Should return status code 200
+    And Sets a "DELETE" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Amet015) [inline trello/api/features/functional/CardFuntional.feature]: add new line here

**Review by Amet015 — APPROVED** [review]

**Pull request merged**